script/tests

```diff
@@ -74,9 +74,11 @@ sources_the_parent_from_child() {
 does_not_source_when_already_sourced() {
   touch $test_dir/.bash_project
 
-  local output="$(test_cd / && test_cd $test_dir && test_cd $test_dir)"
-
-  equals "$output" "projector: sourced $test_dir/.bash_project"
+  {
+    test_cd /
+    test_cd $test_dir
+    test_cd $test_dir
```

Comment:
Why do you change directory to `test_dir` twice?

Reply:
The first cd should result in sourcing the bash_project file. The second one should not as the projector path has not changed. That's what this test confirms.